src/tools/main.cc

```diff
@@ -20,11 +20,15 @@
 #include "common/platform.h"
 
 #include <cassert>
+#include <cstdlib>
+#include <cstring>
 #include <iostream>
+#include <ostream>
 #include <limits.h>
 #include <unistd.h>
 #include <stdio.h>
```

Comment:
As a cosmetic suggestion, consider changing the order of the imports here to be alphabetically ordered.